ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master__4.12.yaml

```diff
@@ -138,7 +106,7 @@ tests:
           cpu: 1000m
           memory: 512Mi
     workflow: optional-operators-ci-gcp
-- as: operator-e2e-aws-periodic-slack
```

Comment:
removed slack from name because we are not using slack notification, but it easy to add https://docs.ci.openshift.org/docs/how-tos/notification/